### PR TITLE
Reunion

### PR DESCRIPTION
**Pull request by dbarrosop**
Proposal for the reunification post.

### CODE REVIEW

File: _posts/2017/2017-07-19-reunification.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+layout: "post"
+tags: ["news"]
+---
+
+Having an individual repo for each driver is causing a lot of administrative overhead so we have decided to merge all repos into a single one. This has some implications you may have to address before and after this happens so I am going to try summarize them here:
+
+1. If you are **installing napalm fully** via the `napalm` package, ie `pip install napalm`:
+    1. Before the merge make sure you pin the version to `<2`. You can easily do that by updating your `requirements.txt` with the string `napalm<2`. If you don't have a requirements file `pip install "napalm<2"` will do the trick. Doing this will ensure nothing breaks for you.
+    1. After the merge, you will have to remove the `<2` constrain and adapt a few import paths. Don't worry, a few import paths is everything that will change. We will provide instructions to make this transition smoother.
+1. If you are **installing napalm partially by installing individual drivers** via their respective packages, i.e. `napalm-eos`, `napalm-junos`, etc...:
```
Comment (mirceaulinic) [inline _posts/2017/2017-07-19-reunification.md]: If users won't be able to do `import napalm_ios` or `import napalm_junos` etc., we have to let them know.

File: _posts/2017/2017-07-19-reunification.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Reunification"
+layout: "post"
+tags: ["news"]
+---
+
+Having an individual repo for each driver is causing a lot of administrative overhead so we have decided to merge all repos into a single one. This has some implications you may have to address before and after this happens so I am going to try summarize them here:
```
Comment (mirceaulinic) [inline _posts/2017/2017-07-19-reunification.md]: This should be formulated like an RFC, explicitly asking for their feedback rather than an imperative "this is what we're going to do and you have to accept"

**Review by mirceaulinic — CHANGES_REQUESTED** [review]
I understand this is far from the version we'll publish - pointed out two major details we should bear in mind.

File: _posts/2017/2017-07-19-reunification.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Reunification"
+layout: "post"
```
Comment (ogenstad) [inline _posts/2017/2017-07-19-reunification.md]: All of the items under the _posts directory will default to `layout: "post"` so that tag won't be needed. Though place add a `description` tag.

File: _posts/2017/2017-07-19-reunification.md (hunk 1 above)
Comment (dbarrosop) [reply _posts/2017/2017-07-19-reunification.md]: They will but they will have to eventually migrate as I tried to explain on the next two subpoints. If you think they are not clear feel free to let me know how you would rephrase. Note this is just informational. We can't be more explicit or give clear instructions until we have a plan and AFAICT today we don't yet have any :P

File: _posts/2017/2017-07-19-reunification.md (hunk 2 above)
Comment (dbarrosop) [reply _posts/2017/2017-07-19-reunification.md]: Ok, will create a github issue for the purpose of people commenting about this and link it in the post so they can raise concerns and provide feedback. However, the decision is ultimately ours as it is us who have to deal this on a daily basis. If people wants to have a greater influence and have a "vote" they should "apply" for the council.

File: _posts/2017/2017-07-24-reunification.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+* **D-Day+14, first version of `napalm`** is released. Instructions to ensure a simple migration will be provided too.
+
+As we want to make sure people has time to update their requirements file to pin versions we have decided to set **D-Day** to **October 20th 2017**.
+
+RFC
+===
+
+As previously stated, this is a rough plan, if you have concerns, comments, suggestions there a few ways you can reach to us to make them known:
+
+1. We have create an [issue](https://github.com/napalm-automation/napalm/issues/392) on github.
```
Comment (dbarrosop) [inline _posts/2017/2017-07-24-reunification.md]: created*

File: _posts/2017/2017-07-24-reunification.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+2. You can also reach us on slack. For details about how to reach us on slack visit the [community](/community) section.
```
Comment (dbarrosop) [inline _posts/2017/2017-07-24-reunification.md]: 2. You can also reach us on slack. Visit [community](/community) for details.
